data.sql

```diff
@@ -5,7 +5,7 @@ Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weigh
 
 insert into animals (name,date_of_birth,weight_kg,neutered,escape_attempts)
  values 
-('Agumon','Feb 3, 2020',10.23,true,0),
+('Agumon','Feb 3, 2020',10.23,true,0) ,
```

Comment:
I understand that you have added the inventory listed below but your data.sql file should have to following for record-keeping purposes

-  ('Charmander', '2020/2/8', '0', 'false', '-11.0');
-    ('Plantmon', '2021/11/15', '2', 'true', '-5.7');
-    ('Squirtle', '1993/04/02', '3', 'false', '-12.13');
-   ('Angemon', '2005/06/12', '1', 'true', '-45.0');
-  ('Boarmon', '2005/06/07', '7', 'true', '20.4');
-  ('Blossom', '1998/10/13', '3', 'true', '17.0');
-  ('Ditto', '2022/05/14', '4', 'true', '22.0');